### PR TITLE
add destination fee balance to xcm sdk docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR is an extension of the one @mmaurello opened: #727 
It also adds info on `destinationFeeBalance` to the v1 reference page

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira

### After Translation Requirements

- [x] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/interoperability/xcm/xcm-sdk/v1/reference.md [hunk 1]
```diff
@@ -110,15 +110,16 @@ The XCM SDK is based on the premise of defining an asset to transfer and then de
...
+    |          Name           |     Type      |                                                                      Description                                                                       |
+    |:-----------------------:|:-------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------:|
+    |        `balance`        | *AssetAmount* |                                           The balance of the asset being transferred for the source address                                            |
+    |         `chain`         |  *AnyChain*   |                                                              The source chain information                                                              |
+    | `destinationFeeBalance` | *AssetAmount* |                                         The balance of the asset used to pay for fees in the desintation chain                                         |
```
Comment (themacexpert) [inline builders/interoperability/xcm/xcm-sdk/v1/reference.md:117]:
```suggestion
    | `destinationFeeBalance` | *AssetAmount* |                                         The balance of the asset used to pay for fees in the destination chain                                         |
```

File: builders/interoperability/xcm/xcm-sdk/v1/reference.md [hunk 2]
```diff
@@ -110,15 +110,16 @@ The XCM SDK is based on the premise of defining an asset to transfer and then de
...
+    |  `existentialDeposit`   | *AssetAmount* |                                      The existential deposit for the asset being transferred on the source chain                                       |
+    |          `fee`          | *AssetAmount* |                                         The amount of fees for the asset being transferred on the source chain                                         |
+    |      `feeBalance`       | *AssetAmount* |                                             The balance of the asset being transferred on the source chain                                             |
+    |          `min`          | *AssetAmount* | The minimum amount of the asset that should be kept on the source chain, taking into consideration the `existentialDeposit` and `fee` for the transfer |
+    |          `max`          | *AssetAmount* |                                               The maximum amount of the asset that *can* be transferred                                                |
 
 ## Core Methods {: #core-sdk-methods }
```
Comment (themacexpert) [inline builders/interoperability/xcm/xcm-sdk/v1/reference.md:125]: There is a typo on Line 222

**Review by themacexpert — APPROVED** [review]
LGTM 👍

File: builders/interoperability/xcm/xcm-sdk/v1/reference.md (hunk 2 above)
Comment (eshaben) [reply builders/interoperability/xcm/xcm-sdk/v1/reference.md:125]: ty!!

**Pull request merged**